### PR TITLE
Lower various logging statement levels to clean up example printing

### PR DESCRIPTION
**Pull request by hjtran**
The playground examples have tons of info logging that drown out any actual logging that the example might've caused. This is somewhat of a big hit to the playground experience (not just noted by me, but by others that I've been introducing to Beam)

I attempted a fix to change the playground's [log level](https://github.com/apache/beam/pull/29948) but that didn't seem to take. Vlado Djerek also tried to figure out where the logging level is set but was also unsuccessful.

In lieu of a real fix, I just changed or deleted various logging statements that I suspect are probably not all that useful. I suspect especially that the translation prints which really clog up the examples don't offer much useful information.

Here's an example of what running the example looks like right now:
image

### CODE REVIEW

**Comment by hjtran** [comment]
reviewer: @tvalentyn

**Comment by hjtran** [comment]
I left a few statements out because they seemed like they're more important, but I think they do still distract from the examples. If we ever find where the log level is set, I wonder if we should just disable the logging altogether and update any examples that use the logger to just use `beam.Map(print)` or `beam.LogElements` instead

File: sdks/python/apache_beam/runners/worker/statecache.py [hunk 1]
```diff
@@ -231,7 +231,7 @@ class StateCache(object):
   """
   def __init__(self, max_weight):
     # type: (int) -> None
-    _LOGGER.info('Creating state cache with size %s', max_weight)
+    _LOGGER.debug('Creating state cache with size %s', max_weight)
```
Comment (tvalentyn) [inline sdks/python/apache_beam/runners/worker/statecache.py:234]: Having this line can be useful when running on production runners, i'd prefer to keep it.

**Comment by tvalentyn** [comment]
> I attempted a fix to change the playground's https://github.com/apache/beam/pull/29948 but that didn't seem to take. Vlado Djerek also tried to figure out where the logging level is set but was also unsuccessful.

By crashing playground intepreter, I see that the main python entrypoint is located in some generated files, like:
/opt/playground/backend/executable_files/8ff1f92b-37b3-4fd7-9a08-fb8d1768ae5f/8ff1f92b-37b3-4fd7-9a08-fb8d1768ae5f.py

We could try to find a place where that is generated and add a line like `logging.getLogger().setLevel(logging.WARNING)` at the top of the file.

**Comment by hjtran** [comment]
> > I attempted a fix to change the playground's #29948 but that didn't seem to take. Vlado Djerek also tried to figure out where the logging level is set but was also unsuccessful.
> 
> By crashing playground intepreter, I see that the main python entrypoint is located in some generated files, like: /opt/playground/backend/executable_files/8ff1f92b-37b3-4fd7-9a08-fb8d1768ae5f/8ff1f92b-37b3-4fd7-9a08-fb8d1768ae5f.py
> 
> We could try to find a place where that is generated and add a line like `logging.getLogger().setLevel(logging.WARNING)` at the top of the file.
 I attempted this in another [PR](https://github.com/apache/beam/pull/30798), though I don't actually know how to test it. I just removed the log level setting altogether, but we could change to just WARNING. IMO though, playground examples seem like an unlikely place to debug runner issues, and the examples are always easy to run locally anyways if we ever find something amiss.

**Comment by tvalentyn** [comment]
> I just removed the log level setting altogether, but we could change to just WARNING. 

+1 to change to warning instead of removing

**Comment by tvalentyn** [comment]
(closed accidentally)

**Comment by tvalentyn** [comment]
I also don't known how to test playground code. a log level change should be safe to merge.

File: sdks/python/apache_beam/runners/worker/statecache.py (hunk 1 above)
Comment (hjtran) [reply sdks/python/apache_beam/runners/worker/statecache.py:234]: Reverted this

**Review by tvalentyn — APPROVED** [review]

**Pull request merged**